### ISSUE TITLE
Multi-mode beatmapsets only require 2 nominations

### ISSUE THREAD

**Issue by Ephemeralis** [ppy/osu-web#6689]
Currently, a set that has difficulties from all 4 game modes only requires the standard two nominations to proceed into qualified. Logically speaking, shouldn't a hybrid set require 2 nominations per game mode present? If 4 game modes are present, 8 nominations should be required, with enforcement from NAT end to ensure that appropriately qualified BNs are awarding those nominations.

I've asked around as to why this isn't already the case and have been met with a mixture of "the current system doesn't support it" and "it's tradition that it's always been this way". We should address this. The fact that giant hybrid sets are entering Ranked with presumably cover to have less than normal oversight is a huge problem.

**Comment by cl8n** [comment]
"the current system doesn't support it" for the same reason as #5484

**Comment by cl8n** [comment]
should nominators of multiple modes be allowed to choose which modes they're nominating a map for? or just one nominate button to do all of the modes they're allowed to?

**Comment by Ephemeralis** [comment]
> should nominators of multiple modes be allowed to choose which modes they're nominating a map for? or just one nominate button to do all of the modes they're allowed to?

The former. Some hybrid BN may opt to only pass judgment on a particular mode they're qualified in, nothing is requiring them to check every mode if they don't want to.

**Comment by Greenshellus** [comment]
I'm generally open to this, one issue I'm seeing with this however is that people will feel less open towards creating hybrid sets in the first place, as one of the perks actually was to only require 1 BN per mode to confirm a set's readiness. If we make it so it always requires 2 BNs for _each_ mode, it's gonna be much more of a hassle for mappers to manage.

Then again, hybrids should get the same treatment in terms of checking each mode carefully as regular single-mode sets do. As it is now, ranking a big 20 GDs Standard set with a side of 3 half-assed Taiko diffs for example is somewhat easy for the amount of difficulties there are, as only one BN has to look at all the Standard diffs, while the other BN only skims through those three diffs in less than an hour and can call it a day. That's not proper checking of a set, that's dumb.


If we implement this though, I'd like to make a proposal that goes hand-in-hand with it.


Allow Probationary BNs to nominate hybrid sets as well.

The big issue with Probationees and hybrid sets always was that they have to take responsibility for the respective mode they're nominating on their own, while ranking any set requires _at least_ one Full BN to check a set. Which, obviously, works for single-mode sets as two BNs cover the same thing, but not so much for hybrid sets, as you'll only have one BN per mode.

If each mode in a hybrid set requires 2 BNs to check it at all times, then I don't see why Probationees should be exempt from hybrids anymore, as it'll work just like checking a regular single-mode set.


TL;DR: sure, but allow probation BNs to do stuff on hybrids too

**Comment by cl8n** [comment]
the probation hybrid rule only exists because the modes aren't split up like this, that'll go away

**Comment by Ephemeralis** [comment]
This has prompted a temporary freeze on new hybrid mapset adoption from the NAT side of things as listed [here](https://github.com/ppy/osu-wiki/pull/4276#event-3826941001), in essence blocking map rankings until it is addressed. Adding for visibility.

**Comment by nekodex** [comment]
Okay, I'm gonna start digging into this - to begin with, we're gonna (temporarily) just change the nomination count required on hybrid sets to be 2x(game modes in set) and rely on the NAT/BN team(s) to self-police these nominations.

Afterwards (this will take a while longer), we'll associate NAT/BN members with their respective game mode(s) so that they can nominate for specific game mode(s) and then enforce these nominations per game mode.
